src/graphql/resolvers.ts

```diff
@@ -28,23 +28,40 @@ export const resolvers = {
 
       return user;
     },
-    users: async (_: any, args: { numberOfUsers?: number }, context) => {
+    users: async (_: any, args: { input?: UsersInput }, context) => {
       if (!context.id) {
         throw new CustomError('Invalid token.', 401);
       }
 
-      const users = await User.find({
+      if (args.input.numberOfUsers < 0) {
+        throw new CustomError('numberOfUsers can not be negative.', 400);
+      }
```

Comment:
Não vale incluir o 0 tbm?

```suggestion
      if (args.input.numberOfUsers <= 0) {
        throw new CustomError('numberOfUsers must be positive.', 400);
      }
```